mesos-client/src/main/resources/reference.conf

```diff
@@ -33,4 +33,17 @@ akka {
     # outgoing connection might be closed if no call is issued to Mesos for a while
     idle-timeout = infinite
   }
-}
+
+  stream {
```

Comment:
As a library, I feel like we shouldn't rely on putting global configuration here for Akka. We should instead be adding the appropriate catch-all error catching (which will expose exceptions properly).

Reply:
Yeah, that does make sense, and I think you're correct there. I couldn't really figure out why we actually need that increased timeout, something slows down the whole startup sequence, and we should probably clean that up instead of adjusting the timeout.

Reply:
@timcharper I've figured it out: The ConnectionPoolSettings creation does a reverse DNS lookup which is really slow on my macbook, which triggered the timeout. I've added a timeout check there and throw a reasonable error message now, so users of the lib will see something useful.